DragonSlayer.h

```diff
@@ -1,8 +1,14 @@
 #pragma once
 
+#include "Character.h"
+
 struct DragonSlayer : Character
 {
+    DragonSlayer(std::string name_, int hp_, int armor_, int attackDamage_ = 4);
+    const std::string& getName() override;
+    std::string getStats() override;
     void attack(Character& other) override;
 private:
     const std::string name;
-};
\ No newline at end of file
+    std::vector<std::unique_ptr<Item>> attackItems;
```

Comment:
Store your attack item as its proper type `AttackItem`.
Re-read the instructions regarding how many attack items the DragonSlayer is supposed to receive.
Include the necessary files to make this happen.
- [ ] revise